tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -345,25 +361,32 @@ def setUp(self):
         if self.__class__ is HarnessCase:
             return
 
-        logger.info('Setting up')
+        logger.info("Setting up")
         # clear files
-        logger.info('Deleting all .pdf')
+        logger.info("Deleting all .pdf")
         os.system('del /q "%HOMEDRIVE%%HOMEPATH%\\Downloads\\NewPdf_*.pdf"')
-        logger.info('Deleting all .xlsx')
-        os.system(
-            'del /q "%HOMEDRIVE%%HOMEPATH%\\Downloads\\ExcelReport*.xlsx"'
```

Comment:
not sure but is there still have such ExcelReport*.xlsx since R1.4.0?

Reply:
no xls in Downloads directory since R1.4.0.